src/rules/selector-attribute-quotes/__tests__/index.js

```diff
@@ -0,0 +1,145 @@
+import { testRule } from "../../../testUtils"
+import rules from "../../../rules"
+import { ruleName, messages } from ".."
+
+const rule = rules[ruleName]
+
+testRule(rule, {
+  ruleName,
+  config: ["always"],
+  skipBasicChecks: true,
+
+  accept: [ {
+    code: "[title] { }",
+  }, {
+    code: "a[target=\"_blank\"] { }",
+  }, {
+    code: "a[ target=\"_blank\" ] { }",
+  }, {
+    code: "[class|=\"top\"] { }",
+  }, /* {
+    code: "[class |= \"top\"] { }",
```

Comment:
> The resolution can be after the merge

Just so I'm clear. Is the intention here is to merge this PR into into the `master` as-is (and perhaps release)? Then we handle the updating of the `postcss-selector-parser` to whatever version comes out of postcss/postcss-selector-parser#71 as a breaking change (perhaps in `7.0.0` if it's merged in time, otherwise `future-major`)?